network-config-analyzer/NetworkConfig.py

```diff
@@ -343,81 +291,25 @@ def get_policy_type(policy):
         return NetworkConfig.ConfigType.Unknown
 
     @staticmethod
-    def append_policy(policy, policies_map, sorted_policies, ingress_policies, current_config_type):
-        """
-        Append a new policy into current config and determine the updated config type
-        :param NetworkPolicy policy: the policy to append
-        :param dict policies_map: the map of policies by (name,type) to policy objects
-        :param sorted_policies: the sorted list of policies
-        :param list ingress_policies: the list of ingress policies
-        :param NetworkConfig.ConfigType current_config_type: the current config type
-        :return: the new config type after adding the policy
-        :rtype: NetworkConfig.ConfigType
-        """
-        new_config_type = current_config_type
-
+    def append_policy(policy, policies_map, layers_map):
```

Comment:
This really should be a method of PoliciesContainer

Reply:
done